semantic_conventions/resource/host.yaml

```
@@ -3,7 +3,7 @@ groups:
     prefix: host
     type: resource
     brief: >
-        A host is defined as a general computing instance.
+        'A host is defined as a computing instance. Examples: physical server, virtual machine, switch or disk array.'
```

To not overlap with the example section and instead of wrapping in unnecessary quotes, what about this?

```suggestion
        A host is defined as a computing instance. For example, physical servers, virtual machines, or switch or disk arrays.
```

---

I have applied the suggestion, with a small change.